woocommerce/woocommerce
# Adding toolbar to the Image Gallery component

### All Submissions:






### Changes proposed in this Pull Request:

Adding the Image Gallery toolbar.

Closes #34437 .

### How to test the changes in this Pull Request:

1. Run storybook and browse to Image Gallery -> Basic story
2. Enable console and view callback logs while testing basic functions:
     - Drag and drop item
     - Click item to reveal toolbar
     - Toolbar -> move left
     - Toolbar -> move right
     - Toolbar -> drag item by handle
     - Toolbar -> set item as cover image
     - Toolbar -> replace item
     - Toolbar -> remove item
4. Ensure that you cannot drag/drop cover item or move with toolbar
5. Ensure that you cannot move final item forward


### Other information:




### FOR PR REVIEWER ONLY:
